packages/ibm-products/src/components/Decorator/constants.js

```diff
@@ -0,0 +1,40 @@
+/**
```

Comment:
i may have left confusing feedback previously.

this file should be named utils since it has utility functions. if you wanted a constants file to maintain things like `const BENIGN = 'Benign'` this would be the place to do it